### ISSUE TITLE
Documentation of Channels could really use an example

### ISSUE THREAD

**Issue by kshyatt** [JuliaLang/julia#14312]
http://docs.julialang.org/en/latest/manual/parallel-computing/#channels

The documentation even references an example file. It would be very nice to have short snippet examples of using `put`, `take`, `isready`, etc. in this section for people who are not familiar with `Channel`s. An example of using the `Channel` in the `for` loop would also be great!

**Comment by ViralBShah** [comment]
Cc @amitmurthy

**Comment by BryanARivera** [comment]
Agreed. I am trying to figure out how to do this.

```
  for alert in channel
      println("Received Alert: $alert")
  end
```

Is this correct?

**Comment by kshyatt** [comment]
Bumping this if @amitmurthy or maybe @shashi want to take a run at it? (Or do we think the docs are good enough?)

**Comment by tkelman** [comment]
> Or do we think the docs are good enough?

Channels still don't have any code examples, so no

**Comment by JeffBezanson** [comment]
Has been fixed; see #17674
